index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Landing Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<main>
+    <h1 id="title"> Survey Form </h1>
+    <p id="description"> Thank you for taking the time to help us improve the platform </p>
+<section class="form-survey">
+    <form class="survey">
+    <div class="list">
+      <label id="name-label">Name</label> 
+      <input type="text" id="name" required placeholder="Enter Your Name"><br>
+    </div>
+    <div class="list">
+      <label id="email-label">Email Address</label>
+      <input type="email" id="email" required placeholder="Your Email Address"><br>
+    </div> 
+    <div class="list">
+      <label id="number-label">Numbers</label>
+      <input type="number" id="number" min="0" max="500" placeholder="put a number"><br>
```

Comment:
Kindly use the **class** selector instead of the **id** selector if you want to select multiple elements because the id is specific to a single element, and you can select multiple elements using the class selector.